File: Praktik 6/BelajarNestedIf.java
import java.util.Scanner;

public class BelajarNestedIf {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.print("Masukkan umur anda = ");
        int umur = scanner.nextInt();
        if (umur <= 5 && umur >=0) {
            System.out.println("Anda masih balita");
            System.out.print("Masukkan umur orang tua anda = ");
            int umurOrangTua = scanner.nextInt();
            if (umurOrangTua < 15) {
                System.out.println("Sebaiknya meminta bantuan pengasuh anak");
            } else {
                System.out.println("Anda bisa mengasuh anak sendiri");
            }
        } else if (umur > 5 && umur <= 19) {
            System.out.println("Anda sudah remaja");
        } else if (umur > 19) {
            System.out.println("Anda sudah dewasa");
        } else {
            System.out.println("Umur anda tidak valid");
        }
        
    }
}

File: Praktik 6/BelajarNestedIf.py
umur = input("Masukkan umur anda = ")
umur = int(umur)
if (umur >= 0 and umur <=5):
    print("Anda balita")
    umur_orangtua = input("Masukkan umur orang tua anda = ")
    if (int(umur_orangtua) < 15):
        print("Orang tua anda perlu pengasuh")
    else:
        print("Anda dapat mengasuh anak Anda sendiri")
elif (umur > 5 and umur <= 19):
    print("Anda remaja")
elif (umur > 19):
    print("Anda dewasa")
else :
    print("Umur anda tidak valid")